mkosi/__init__.py

```diff
@@ -6251,7 +6244,8 @@ def find_output(args: argparse.Namespace) -> None:
     else:
         return
 
-    args.output_dir.mkdir(parents=True, exist_ok=True)
+    mkdirp_chown_current_user(args.output_dir.parent, args.no_chown)
+    mkdirp_chown_current_user(args.output_dir, args.no_chown)
```

Comment:
Why do we need both here if the function already takes care of creating the parent directories?